### PR TITLE
Florian sandbox

### PR DESCRIPTION
**Pull request by fwmetzler**
I created a new notebook which depicts different parts of the internuclear potential. 

Potential issues with the preview may arise due to the import and use of a dygraph library via files in the dygraphs subfolder. Let's maybe try out and see whether nbviewer and/or Colab can handle this or not.

### CODE REVIEW

File: src/01-nuclear-energetics.md [hunk 1]
```diff
@@ -382,6 +382,7 @@ parents_alpha_decay.head(10)
 parents_alpha_decay.plot.scatter(x="A", y="E_kin (MeV)",figsize=(15,8), 
                                  title="Kinetic energy of alpha decay products     (Fig 3)");
 plt.plot((270, 0), (0, 0), 'r-');
+#plt.xlim(50,100);
```
Comment (mklilley) [inline src/01-nuclear-energetics.md:385]: Looks like you modified nuclear energetics file - did you mean to do this?

File: src/02-internuclear-potentials.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
+<a href="https://colab.research.google.com/github/project-ida/nuclear/blob/master/02-internuclear-potentials.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://nbviewer.jupyter.org/github/project-ida/nuclear/blob/master/02-internuclear-potentials.ipynb" target="_parent"><img src="https://nbviewer.jupyter.org/static/img/nav_logo.svg" alt="Open In nbviewer" width="100"/></a>
+
+
+# Internuclear potentials
+
+
+This notebooks looks at different sections of the internuclear potential between two nuclei, specifically two deuterium nuclei as in the case of a molecule of deuterons D2.
```
Comment (mklilley) [inline src/02-internuclear-potentials.md]: Typo - this notebook

File: src/02-internuclear-potentials.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+```python
+kw = pd.read_csv("./data/k-w_potential.csv")
+```
+
+```python
+def morsepot_au(r): # in atomic units
+    a = 0.4
+    return (1+0.1745)*(np.exp(-2*a*(r-1.4))-2*np.exp(-a*(r-1.4))) 
+    # I had to add +1 to the minimum energy and adjust a accordingly
```
Comment (mklilley) [inline src/02-internuclear-potentials.md]: Why did you need to do this?

File: src/02-internuclear-potentials.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+kw.head()
+```
+
+```python
+plt.scatter(kw["Rpm"],kw["Eev"])
+plt.plot(kw["Rpm"],hartree_to_ev(kw["Morseev"]))
+plt.xlabel("internuclear distance [pm]")
+plt.ylabel("potential energy [eV]")
+plt.title("KW potential from table in paper alongside Morse potential")
+plt.show()
```
Comment (mklilley) [inline src/02-internuclear-potentials.md:78]: Would be nice to have a legend

File: src/02-internuclear-potentials.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+kw_copy.head()
+```
+
+```python
+plt.scatter(kw["Rpm"],kw["Eev"])
+plt.plot(kw_copy["Rpm"],kw_copy["Eev"])
+plt.xlabel("internuclear distance [pm]")
+plt.ylabel("potential energy [eV]")
+plt.title("KW potential from table in paper alongside interpolated KW potential")
+plt.show()
```
Comment (mklilley) [inline src/02-internuclear-potentials.md:105]: Would be nice to have a legend

File: src/02-internuclear-potentials.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+```python
+plt.plot(kw_copy["Rpm"],kw_copy["Eev"])
+plt.plot(Rpm2,hartree_to_ev(Cev))
+
+plt.ylim(-40,100)
+plt.xlim(-0,100)
+plt.xlabel("internuclear distance [pm]")
+plt.ylabel("potential energy [eV]")
+plt.title("Coulomb potential and KW potential")
+plt.show()
```
Comment (mklilley) [inline src/02-internuclear-potentials.md:174]: Would be nice to have a legend

File: src/02-internuclear-potentials.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+```
+
+## Adding a Woods-Saxon nuclear potential
+
+
+The Woods-Saxon (WS) potential is a mean potential that is assumed to reasonably describe the attraction of the nuclear force from a given nucleus with A=number of nucleons.
+
+```python
+A = 4 # 4 nucleons in two deuterium atoms
+V0 = 28.5e6 #50 # ground state energy see for He-4 https://pdfs.semanticscholar.org/4c24/b2d33f3b3ca2f288897967dc3a6fdb51f1a0.pdf
```
Comment (mklilley) [inline src/02-internuclear-potentials.md:194]: What does the number 50 mean?

File: src/02-internuclear-potentials.md [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+plt.xlabel("internuclear distance [pm]")
+plt.ylabel("potential energy [eV]")
+plt.title("Coulomb potential and KW potential")
+plt.show()
+```
+
+## Adding a Woods-Saxon nuclear potential
+
+
+The Woods-Saxon (WS) potential is a mean potential that is assumed to reasonably describe the attraction of the nuclear force from a given nucleus with A=number of nucleons.
```
Comment (mklilley) [inline src/02-internuclear-potentials.md:190]: Would be nice to have a Latex formula like you had for the Morse potential

File: src/02-internuclear-potentials.md [hunk 9]
```diff
@@ -0,0 +1,234 @@
...
+
+```python
+rpm_all = np.concatenate((Rpm1,Rpm2))
+
+fullpotential = pd.DataFrame({'Rpm': rpm_all, 'WSev': 0, 'Cev': 0, 'KWev': 0})
+```
+
+```python
+thisr = fullpotential.loc[fullpotential["Rpm"] < 1,"Rpm"]
+thisr
```
Comment (mklilley) [inline src/02-internuclear-potentials.md]: `thisr ` - I think you can delete that

File: src/02-internuclear-potentials.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+thisws = woodssaxon_ev(thisr*1e-12, V0, R, a)
+thisws
```
Comment (mklilley) [inline src/02-internuclear-potentials.md]: `thisws` - I think you can remove this

File: src/02-internuclear-potentials.md [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+thisr
+thisc = hartree_to_ev(energyau(1,1,pm_to_bohr(thisr)) - 1.9037)
+thisc
+
+fullpotential.loc[thisc.index,"Cev"] = thisc
+```
+
+```python
+plt.plot(fullpotential["Rpm"],fullpotential["WSev"]+fullpotential["Cev"])
+plt.xlim(0,0.01)
```
Comment (mklilley) [inline src/02-internuclear-potentials.md:245]: Why is there a discontinuity in the graph?

File: src/02-internuclear-potentials.md [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+
+# Generate data for the plot
+x = fullpotential["Rpm"]
+y = [fullpotential["assembled"]]
+
+# Plot the data on the figure
+fig.plot(x,y)
+
+# Show the figure in this cell of the notebook
+fig.show()
```
Comment (mklilley) [inline src/02-internuclear-potentials.md]: Cannot see the graph

**Review by mklilley — CHANGES_REQUESTED** [review]
Lovely to see this all in a nicely presented notebook. Just a few changes will make it ready I'd say.

File: src/02-internuclear-potentials.md [hunk 13]
```diff
@@ -0,0 +1,234 @@
+<a href="https://colab.research.google.com/github/project-ida/nuclear/blob/master/02-internuclear-potentials.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://nbviewer.jupyter.org/github/project-ida/nuclear/blob/master/02-internuclear-potentials.ipynb" target="_parent"><img src="https://nbviewer.jupyter.org/static/img/nav_logo.svg" alt="Open In nbviewer" width="100"/></a>
```
Comment (mklilley) [inline src/02-internuclear-potentials.md]: `nuclear/blob` should be `nuclear-reactions/blob`

File: src/02-internuclear-potentials.md (hunk 3 above)
Comment (fwmetzler) [reply src/02-internuclear-potentials.md]: I'm not sure what determines the absolute values of the floating potentials. I added some more comments and for now I'll just say that I had to do it to match the KW potential to the formula given in KN

File: src/01-nuclear-energetics.md (hunk 1 above)
Comment (fwmetzler) [reply src/01-nuclear-energetics.md:385]: Thanks for pointing this out. I didn't meant to do this.

File: src/02-internuclear-potentials.md (hunk 4 above)
Comment (fwmetzler) [reply src/02-internuclear-potentials.md:78]: Added it for the next iteration

File: src/02-internuclear-potentials.md (hunk 5 above)
Comment (fwmetzler) [reply src/02-internuclear-potentials.md:105]: Added it for the next iteration

File: src/02-internuclear-potentials.md (hunk 8 above)
Comment (fwmetzler) [reply src/02-internuclear-potentials.md:190]: Yes, great point. I added that and a bit more explanation

File: src/02-internuclear-potentials.md (hunk 9 above)
Comment (fwmetzler) [reply src/02-internuclear-potentials.md]: done

File: src/02-internuclear-potentials.md (hunk 10 above)
Comment (fwmetzler) [reply src/02-internuclear-potentials.md]: done

**Comment by fwmetzler** [comment]
I implemented the suggested changes and will start a new pull request

**Pull request closed without merging**